src/components/Auth/SignupForm.tsx

```diff
@@ -13,8 +13,11 @@ interface SignupFormProps {
 function SignupForm(props: SignupFormProps) {
   const { signupInfo, handleOnChange, handleComplete } = props;
   const isPasswordEqual = signupInfo.password.value === signupInfo.passwordConfirm.value;
-
   const InfoList = Object.keys(signupInfo);
```

Comment:
```suggestion
  const InfoList = Object.keys(signupInfo).slice(0, -1).filter(isKeyOfSignUpInfo);
```

이렇게하면 컴포넌트의 return 부분에서 map 돌릴 때 if문을 제거할 수 있을 것 같아유!!